docs/guides/build-wallet.md

```diff
@@ -189,10 +112,12 @@ async function init () {
   })
 }
 
-init().then(_ => console.log('Wallet initialized!'))
+init()
+console.log('Wallet initialized!')
```

Comment:
wallet initialisation still depends on async account adding, so I'm reverting this change